File: php/tests/fixtures/AssessorStub.php
<?php

declare(strict_types=1);

namespace FactChecker\Tests\fixtures;

use FactChecker\AssessService\Assessor;

class AssessorStub implements Assessor
{
    public function getScore(string $sentence): int
    {
        return 4;
    }

    public function getOpinion(string $sentence): string
    {
        return 'believable';
    }
}

File: java/app/src/test/java/factchecker/fixtures/AssessorStub.java
package factchecker.fixtures;

import factchecker.AssessService.Assessor;

public class AssessorStub implements Assessor {
    @Override
    public Integer getScore(String sentence) {
        return 4;
    }

    @Override
    public String getOpinion(String sentence) {
        return "believable";
    }
}
